### PR TITLE
Editing in a few minor changes in the documents

### PR DESCRIPTION
**Pull request by malvikasharan**
Reviewing documents and adding a few minor points that could be useful
- [x] before
- [x] during
- [x] after

### CODE REVIEW

**Comment by malvikasharan** [comment]
@tobyhodges These documents are really good and very comprehensible. I have gone through the three documents mentioned in my PR and reviewed them keeping my book-dash event in mind. I have reordered a few points to keep them in chronological order, which may make it look like I have done a lot of edits (but it is not true :D). Also, I have added space between each point to make it readable.

I know that you all are already creating templates, which will be very useful to link to the different documents (planning template, primer/onboarding template, link to introductory GitHub lesson to share with participants, shared document for the event, feedback form/questions, post-event briefing document and report template). I have many of these listed here which can be reused I hope: https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash/organisers_files-2020.

Additionally, I wonder if a document with different formats for the sprint can be supplied to allow people to choose their event format.

File: before.md [hunk 1]
```diff
@@ -17,45 +18,59 @@
...
-  **[Online]** For sprints running online with the potential for participants from multiple
-  regions/time zones,
-  it is vital to announce time(s) and date(s) for your sprint in a way that
-  allows visitors to see the equivalent in their local time.
-  Always list event timings in UTC and use tools like [timeanddate.com](https://timeanddate.com) to provide automatic conversion to local time.
-- **Block time in your calendar _after_ the sprint** for wrap-up/follow-up tasks.
+    - Conversely, you may want to avoid clashes with another event if your team is going to be engaged elsewhere or have other commitments.
+    - Announce the dates for your sprint at least 6 weeks in advance. 
+
+- **Add dates and time in your calendar**: In addition to the event dates, schedule a few planning meetings with the committee members _before_ the sprint and one meeting _after_ the sprint for wrap-up/follow-up tasks.
```
Comment (tobyhodges) [inline before.md]:
```suggestion
- **Add dates and time in your calendar**: In addition to the event dates, schedule a few planning meetings with the committee members _before_ the sprint and one meeting/co-working session _after_ the sprint for wrap-up/follow-up tasks.
```

File: after.md [hunk 2]
```diff
@@ -1,36 +1,43 @@
...
+
 - **Contribute to these recommendations.**
   While organising and running your sprint, did you find any useful resources or steps not already covered
   in this guide?
   Do you have any advice or guidance for others who may follow these recommendations in future?
   If so, please contribute back to improve this resource!
   [The recommendations are maintained in this GitHub repository](https://github.com/tobyhodges/lesson-sprint-recommendations).
-  To tell us about changes you would like to see to these recommendations,
-  please email the authors (tobyhodges at carpentries dot org),
+  To tell us about changes you would like to see these recommendations,
```
Comment (tobyhodges) [inline after.md]:
```suggestion
  To tell us about changes you would like to see to these recommendations,
```

Adding this "to" back in.

File: before.md [hunk 3]
```diff
@@ -17,45 +18,59 @@
...
     See [During the Sprint](/during.md) for more details.
     - Mailing lists for initial organisation and other important and more permanent communication, e.g. scheduling meetings.
     - **[Online]** Video conferencing technology that is an accessible choice for everyone and suitable for the format of the sprint.
-    - Collaborative note taking platform suitable for recording and communicating decisions, minutes and actions from meetings.
-- **Read about good practices in lesson design.** To make sure your lesson is as effective as possible,
-  consult [The Carpentries Curriculum Development Handbook](https://cdh.carpentries.org)
-  and think about how you and your collaborators will follow those recommendations when creating the lesson material during the sprint.
+    - Decide on the basic etiquette for participation, such as how to collect ideas, who to contact for support, how to participate in an ongoing discussion, how to contribute to content development and how to review contributions. 
+    - Collaborative note-taking platform is suitable for recording and communicating decisions, minutes and actions from meetings.
```
Comment (tobyhodges) [inline before.md]:
```suggestion
    - Choose a collaborative note-taking platform for recording and communicating decisions, minutes and actions from meetings. (See [the tools section](tools.md) for details of some different note taking platforms.)
```

File: before.md [hunk 4]
```diff
@@ -76,13 +91,13 @@ There is a number of tools that in combination could be used for running effecti
...
   that could be tackled during the event.
   [GitHub Projects](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-project-boards)
   can help with this,
-  allowing you to tag particular issues within scope for the sprint
+  allowing you to label particular issues within scope for the sprint
   and even organise them further into categories to help contributors find something to work on.
   Even if you cannot invest time into managing issues to this extent,
   make sure open issues include all the contextual information necessary
-  for a new or part-time contributor to understand what needs doing
+  for a new or part-time contributor to understanding what needs doing
```
Comment (tobyhodges) [inline before.md]:
```suggestion
  for a new or part-time contributor to understand what needs doing
```

File: during.md [hunk 5]
```diff
@@ -100,37 +107,42 @@
...
+
+- **Flag pull requests as a draft and ready for review as you work.**
   Opening draft pull requests early, while work on a task is ongoing,
   can help contributors understand where progress is being made and identify
   tasks/issues that still need to be tackled.
   Marking pull requests as ready for review helps maintainers prioritise their
   focus during the sprint and maintains momentum by ensuring that completed work
   is merged into the lesson as soon as possible.
-  Opening draft pull requests is particularly important at the end of a sprint,
+  Opening draft pull requests are particularly important at the end of a sprint,
```
Comment (tobyhodges) [inline during.md]:
```suggestion
  Opening draft pull requests is particularly important at the end of a sprint,
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Thank you for this thorough review, and the very good points you added @malvikasharan! I have suggested only a few minor changes.

File: during.md (hunk 5 above)
Comment (malvikasharan) [reply during.md]: I see I made a lot wrong edits :D Thank you for being patient with this.

File: during.md [hunk 6]
```diff
@@ -1,51 +1,60 @@
...
+  You could run this primer session a week or two days before the sprint begins.
+  This can also be the first session of the sprint event before co-working sessions begin.
+
+- **Divide co-working sessions with scheduled check-ins** 
+  Plan your sprint to have several co-working sessions that are followed by check-ins or report outs to allow contributors to reflect on their progress.
+  This will also allow them to share what they've achieved, what's blocking their progress, where they would like some feedback and what they intend to do next.
+  We recommend using a web-based shared clock such as [cuckoo.team](https://cuckoo.team/) to keep track of co-working, share-out and breaks.
+
+- **Record information about participants and contributions.**
+  - Start your event by asking people to sign-in to the shared document by providing names, Orcid identifiers, GitHub usernames, and any other identifiers/handles that can be used to acknowledge their contributions.
```
Comment (malvikasharan) [inline during.md]:
```suggestion
  - Start your event by asking people to sign-in to the shared document by providing names, [ORCID identifiers](https://orcid.org/), [GitHub usernames](https://github.com), and any other identifiers/handles that can be used to acknowledge their contributions.
```

**Comment by tobyhodges** [comment]
Thanks so much! @all-contributors please add @malvikasharan for review and code

**Pull request merged**